README.md

```diff
@@ -54,6 +54,65 @@ demo.rb -- 8 warnings:
   [3]:Dirty#awful has unused parameter 'y' (UnusedParameters)
 ```
 
+## Fixing Smell Warnings
+
+Since `reek` mostly focuses on high-level code smells we can't tell you how to fix warnings in
+a generic fashion since this is and will always be completely dependent on your domain
+language and bussiness logic.
+
+A simple example though might help you get going. Have a look at the sample code below using a
+"Ruby on Rails" model (be aware though that this example is truncated and not really working code):
+
+```Ruby
+class ShoppingCart < ActiveRecord::Base
+  has_many :items
+
+  def gross_price
+    items.sum {|item| item.net + item.tax }
+  end
+end
+
+class Item
+  belongs_to :shopping_cart
+end
+```
+
+Running `reek` on this file like this:
+
+```
+reek app/models/shopping_cart.rb
+```
+
+would report:
+
+```
+  [5, 5]:ShoppingCart#gross_price refers to item more than self (FeatureEnvy)
+```
+
+Fixing this is pretty straightforward. Put the gross price calculation for a single item
+where it belongs, which would be the `Item` class:
+
+```Ruby
+class ShoppingCart < ActiveRecord::Base
+  has_many :items
+
+  def gross_price
+    items.sum {|item| item.gross_price }
```

Comment:
>> items.sum(&:gross_price) would drive this point a bit more IMHO

I disagree. I picked this style deliberately to not confuse beginners with rubys symbol2proc (which is incredibly hard to google since you dont know how this is called).